hive-hooks/path-conversion-hook/src/test/java/com/expediagroup/apiary/extensions/hooks/pathconversion/config/ConfigurationTest.java

```diff
@@ -0,0 +1,81 @@
+/**
+ * Copyright (C) 2018-2021 Expedia, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.expediagroup.apiary.extensions.hooks.pathconversion.config;
+
+import static java.lang.String.format;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Mockito.when;
+
+import java.util.Properties;
+
+import org.apache.hadoop.hive.conf.HiveConf;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+
+@RunWith(MockitoJUnitRunner.class)
+public class ConfigurationTest {
+
+    @Mock HiveConf hiveConf;
+
+    @Test
+    public void shouldCheckHiveSiteForPathReplacementEnablement() {
+        Properties mockProps = new Properties();
```

Comment:
```suggestion
        Properties properties = new Properties();
```
You could also consider making this a member variable so you only have to declare it once as every test uses this.